presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -139,3 +140,36 @@ def test_remove_analyzer(self):
                                      language='en')
 
         assert len(res) == 0
+
+    def test_Apply_with_language_returns_correct_response(self):
+        analyze_engine = AnalyzerEngine(MockRecognizerRegistry())
+
+        request = AnalyzeRequest()
+        request.analyzeTemplate.languageCode = 'en'
```

Comment:
I don't understand how do you test the language?
You seem to define it, but how do you test the logic?

Reply:
This is just setting up the request. Then, we call Apply(request,None) which is the function that's being used by the service. There's an additional test that shows it breaks if you don't define any language.

Reply:
I changed the test to just test the AnalyzerEngine.Apply and not the inside recognizers.